src/__tests__/restLink.ts

```diff
@@ -97,6 +98,106 @@ describe('Configuration', () => {
       expect(data.post.title).toBeDefined();
       expect(data.post.tags[0].name).toBeDefined();
     });
+    // it('fieldNameNormalizer Too Late graphql-anywhere issues/2744', async () => {
```

Comment:
Maybe use the [skip method from jest](https://facebook.github.io/jest/docs/en/api.html#testskipname-fn) ?

Reply:
Thanks for the link! I didn't know about that feature. Doing that.